src/R/Components/Impl/Containers/Implementation/ContainerManager.cs

```diff
@@ -82,7 +82,7 @@ internal class ContainerManager : IContainerManager {
             }
         }
 
-        public async Task<IContainer> CreateLocalDockerAsync(string name, string username, string password, string version, CancellationToken cancellationToken = default(CancellationToken)) {
+        public async Task<IContainer> CreateLocalDockerAsync(string name, string username, string password, string version, int port, CancellationToken cancellationToken = default(CancellationToken)) {
             if (string.IsNullOrWhiteSpace(version)) {
                 version = "3.4.1";
```

Comment:
Hm, why it is hardcoded? We can get latest from interpreter list perhaps? The list is local, but gives an idea what user is working with. Also, we could prepopulate with that number.

Reply:
It should be "latest". I will update this.